### ISSUE TITLE
Compatibility with normal multilayer perceptron

### ISSUE THREAD

**Issue by masakinakada** [pplonski/keras2cpp#15]
Hi @pplonski Thank you so much for this amazing keras converter.
I have MLP trained using keras and want to use in C++.

Is this working with MLP which has relu and linear as an activation function? 
I noticed that you mentioned you focused on CNN.

Thank you very much in advance.
Masaki

**Comment by pplonski** [comment]
Hi! Right now, there is softmax and Relu activation implemented. Adding linear activation should be easy. Would you add it? In function `keras::LayerActivation::compute_output`

**Comment by masakinakada** [comment]
Yes I added it! As you said, linear layer was easy.
Just wanted to make sure other parts are fine too!

Thanks you @pplonski !

**Comment by pplonski** [comment]
Great! Would you like to pull your changes?

**Comment by masakinakada** [comment]
push my change?

**Comment by pplonski** [comment]
Sure, please make pull request.
